### ISSUE TITLE
Question about loss function

### ISSUE THREAD

**Issue by jiali1025** [atomistic-machine-learning/schnetpack#540]
Sorry for bothering you! 

I want to know how you implement the loss in the original paper which has both energy loss and the differentiation of energy to get force. I see in the original tf repo it is calculated via Ep = model(*atoms_input)
        Fp = -tf.gradients(tf.reduce_sum(Ep), atoms_input[1])[0]

However, I didn't find this loss in this repo, maybe I overlooked it or there are some reasons to not use it.

Thanks !

**Comment by jnsLs** [comment]
Hello, 

in schnetpack 2.0 the loss function is defined in ```schnetpack.task.AtomisticTask```. More precisely, we define a loss_fn for each ```schnetpack.task.ModelOutput``` and the method ```loss_fn``` in ```schnetpack.task.AtomisticTask``` builds a combined loss for all targets. 
I hope that helps.

Kind regards, Jonas
